src/main/webapp/style.css

```diff
@@ -1,8 +1,14 @@
 h1 {
-  font-family: Verdana, sans-serif; 
+  font-family: Verdana, sans-serif;
   font-size: x-large;
 }
 
-#emphasis {
+.emphasis,
+a {
   color: violet;
 }
+
+a.active,
+a:hover {
+  color: rebeccapurple !important;
```

Comment:
Just wondering why we need the !important?

Reply:
The color wasn't showing up without it for some reason. I think the bootstrap styling was overriding it.